providers/route53.go

```diff
@@ -21,6 +21,21 @@ var (
 )
 
 func init() {
+	if len(os.Getenv("AWS_REGION")) == 0 {
```

Comment:
Nit: as we are evaluating all env vars here, the check below is no longer needed:

https://github.com/rancher/external-dns/blob/a76ffa3d3804e926c4399504bf368311a4566e9a/providers/route53.go#L44